test/unittests/skills/test_skill_manager.py

```diff
@@ -186,14 +186,3 @@ def test_activate_skill(self):
 
         self.skill_manager.activate_skill(message)
         test_skill_loader.activate.assert_called_once_with()
-
-    def test_reload_modified(self):
```

Comment:
Could this test be replaced, mocking `skill.reload` and checking for 1x call and `self.last_modified` updates